runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -277,9 +287,21 @@ public void render(Graphics2D graphics, final OverlayLayer layer)
 					if (inOverlayDraggingMode)
 					{
 						final Color previous = graphics.getColor();
-						graphics.setColor(movedOverlay == overlay ? MOVING_OVERLAY_ACTIVE_COLOR : MOVING_OVERLAY_COLOR);
+						final Stroke previousStroke = graphics.getStroke();
+
+						if (hoveredOverlay == overlay && !clientUI.getCurrentCursor().equals(clientUI.getDefaultCursor()))
```

Comment:
this should probably be a reference comparison, or better yet, a flag for the resizing mode rather than reusing the cursor